docs/DIRECTIVES.md

```
@@ -0,0 +1,144 @@
+
+## Directives
+
+| Directive | Description | Context |
+|---|---|---|
+| [keyval](#keyval) | Define a variable from key-value pairs | http, stream |
+| [keyval_zone](#keyval_zone) | Define a shared memory zone | http, stream |
+| [keyval_zone_redis](#keyval_zone_redis) | Define a Redis zone | http, stream |
+
+### keyval
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)